src/components/FAQModal.html

```diff
@@ -125,7 +125,40 @@ <h5>Frequently Asked Questions</h5>
             Roamer Modifier: <code>50×</code><br/>
             Repeatball Modifier: <code>5×</code></span></p>
             <p><strong>What is the EV Attack Bonus?</strong><br/>
-            <span class="text-muted">The EV bonus is multiplicative, and scales linearly until 1.5x (50EVs). The bonus thereafter has diminishing returns relative to the number of EVs gained, and maxes at 2x (1770EVs).</span></p>
+            <span class="text-muted">The EV bonus is multiplicative, and scales linearly until 1.5x (50EVs). The bonus thereafter has diminishing returns relative to the number of EVs gained. EVs can be gained indefinitely. </span></p>
+            <p><strong>What are the EV damage multipliers?</strong><br/>
+            <span class="text-muted">
+                <style>table, th, td {border: 1px solid black;border-collapse: collapse;}
```

Comment:
This affects all other tables. You either need to use the html attribute `style=""` on the table, or give the table a class and add the css onto that class (in a css file)

Reply:
I would rather we don't manually adjust the style too much (especially colors) as otherwise it won't change with whichever theme the user has selected.